agent/pkg/crane/k8s/configmap.go

```diff
@@ -51,7 +51,7 @@ func (cm *configmap) deployConfigMapData(namespace, name string, envList map[str
 		return err
 	}
 
-	result, err := client.Apply(context.TODO(),
+	result, err := client.Apply(context.Background(),
```

Comment:
We could use the configmap.ctx context. Here and other facade-like structs too.

Reply:
True, I didn't notice that field 👀